eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -204,8 +204,9 @@ stages:
       - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
       - download: current
         artifact: ${{parameters.ArtifactName}}-signed
-      - task: MSBuild@1
-        displayName: 'Publish to blobfeed'
+      - task: NuGetCommand@2
+        displayName: 'Publish to DevOps Feed'
         inputs:
-          solution: '$(AzureSDKBuildToolsPath)/tools/blobfeedtool/BlobFeedPublishHelper.proj'
```

Comment:
For at least some transition period can you include both the blobfeed and the devops feed? It allows folks time to move to the devops feed.